### PR TITLE
add plugin more satisfaction + transifex

### PR DESCRIPTION
**Pull request by Otijom**

### CODE REVIEW

File: source/moresatisfaction/index.rst [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+   .. figure:: images/Satisfaction-2.png
+      :alt:
+
+Create a question
+-----------------
+
+- Enter your survey and go to the **question** tab
+- Click on Add a question
+- Select the type of question **yes/no** and enter the text that suits you
```
Comment (cconard96) [inline source/moresatisfaction/index.rst:47]:
```suggestion
- Enter your survey and go to the **Question** tab
- Click on Add a question
- Select the type of question **Yes/No** and enter the text that suits you
```

File: source/moresatisfaction/index.rst [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+- Then click **Add**
+
+.. figure:: images/Satisfaction-3.png
+   :alt:
+
+Create a text field
+-------------------
+
+- In the question tab, click on **Add a question**
+- select the **text** type
```
Comment (cconard96) [inline source/moresatisfaction/index.rst:57]:
```suggestion
- Select the **Text** type
```

File: source/moresatisfaction/index.rst [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+.. figure:: images/Satisfaction-4.png
+   :alt:
+
+Create a note
+-------------
+
+- In the question tab, click on **Add a question**
+- select the **note** type
+- You can select a score up to 10 and set a default value
+- Click on Add when you are happy with your settings
```
Comment (cconard96) [inline source/moresatisfaction/index.rst:69]:
```suggestion
- select the **Note** type
- You can select a score up to 10 and set a default value
- Click on "Add" when you are happy with your settings
```

**Review by cconard96 — CHANGES_REQUESTED** [review]

File: source/moresatisfaction/index.rst [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+Install the plugin
+------------------
+
+.. info::
+   A `basic licence <https://services.glpi-network.com/#offers>`__ (or higher) is required. This plugin is also available on `Cloud <https://glpi-network.cloud/fr/>`__.
+
+-  Go to the marketplace. Download and install the plugin **More satisfaction**
+
+.. figure:: images/Satisfaction.png
```
Comment (trasher) [inline source/moresatisfaction/index.rst:21]: Image seesm missing

File: source/moresatisfaction/images/Satisfaction-2.png
Comment (trasher) [inline source/moresatisfaction/images/Satisfaction-2.png:1]: Why "Entité racine" only is in french? All should be in english

**Review by trasher — CHANGES_REQUESTED** [review]

File: source/moresatisfaction/index.rst (hunk 4 above)
Comment (Otijom) [reply source/moresatisfaction/index.rst:21]: It's ok for the picture

File: source/moresatisfaction/images/Satisfaction-2.png
Comment (Otijom) [reply source/moresatisfaction/images/Satisfaction-2.png:1]: no idea! it's been modified anyway

**Pull request merged**